azure-iot-device/tests/common/pipeline/test_pipeline_flow.py

```diff
@@ -0,0 +1,216 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
>-------------------------------------------------------------------------- [](start = 2, length = 74)

CodeFlow doesn't recognize this as a rename, but github does.  If you look at the second commit for this PR on the github site, you can see the changes to this file.